code/graphics/render.cpp

```diff
@@ -1054,7 +1054,7 @@ void gr_rect(int x, int y, int w, int h, int resize_mode, float angle) {
 		float offsetX = x + w / 2.0f;
 		float offsetY = y + h / 2.0f;
 		path->translate(offsetX, offsetY);
-		path->rotate(fl_radians(angle));
+		path->rotate(angle);
```

Comment:
I searched through the codebase for `gr_rect` and it looks like no other functions are affected beyond the scripting ones, which is good.  But you should change the documentation for the `gr_rect` declaration (in render.h).